lib/DDG/Spice/SimilarSites.pm

```diff
@@ -15,17 +16,20 @@ topics "everyday", "special_interest";
 attribution github => ['https://github.com/Adman', 'Adman'],
             twitter => ['http://twitter.com/adman_X', 'adman_X'];
 
-triggers startend => "similar sites", "similar sites to", "similar web to", "sites like", "websites like", "webs like", "pages like";
+triggers start => ["sites like", "site like", "websites like", "website like", "webs like", "pages like"];
+
+triggers any => ["similar sites", "similar web", "websites similar", "website similar", "similar websites", "similar website"];
 
 spice to => 'http://www.similarsitesearch.com/api/similar/$1';
 
 spice wrap_jsonp_callback => 1;
 
-handle remainder => sub {
-    s/^https?:\/\///;
-
+handle remainder_lc => sub {
+    $_ =~ s/(to|like|\s)*//g;
+    $_ =~ s/^https?:\/\///;
     # remove whitespace
-    s/^\s+|\s+$//g;
+    trim($_);
+
```

Comment:
Ah, well it's best if we can avoid making redundant API calls because the assumption is that if we triggered an IA, we're expecting it to return a result.